lib/msf/core/handler/reverse_tcp_double.rb

```diff
@@ -95,16 +95,22 @@ def start_handler
       sock_inp = nil
       sock_out = nil
 
-      print_status("Started reverse double handler")
+      unless datastore['MODULEOWNER'] == Msf::Exploit::Remote::BrowserAutopwnv2
```

Comment:
Adding if cases around each of the handler print statements doesn't make much sense. Maybe BAP should just change the output handle of the sub-modules to a buffer and selecting show/save this text?